### PR TITLE
PR for BXMSDOC-6596: Document introduction of emails in Tasks notifications based on BAPL-1595, BAPL-1684.

### PR DESCRIPTION
**Pull request by kaldesai**
**JIRAs:**
[Epic](https://issues.redhat.com/browse/BXMSDOC-6324)
[Dedicated JIRA](https://issues.redhat.com/browse/BXMSDOC-6596)

**Doc preview:**
[II. Interacting with processes and tasks](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-6596-RHPAM-EmailTaskNotification/#configuring-emails-in-task-notification-proc)

**Doc impact:**
Please check chapter **16. Configuring emails in task notification** in the II. Interacting with processes and tasks document.

### CODE REVIEW

File: doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
+[id='configuring-emails-in-task-notification-proc']
+
+= Configuring emails in task notification
+
+You can directly add any email addresses to send task notification emails to the users and groups.
```
Comment (elguardian) [inline doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc:5]: I would reword this. Prior to the change you could send notifications to users and/or groups. Now with the change you can send notifications to users and/or groups and/or emails addresses

Comment (kaldesai) [reply doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc:5]: Okay! I will restructure the sentence.

**Review by elguardian — APPROVED** [review]

File: doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
+[id='configuring-emails-in-task-notification-proc']
+
+= Configuring emails in task notification
+
+Earlier it was possible to send notifications only to users or group of users in {CENTRAL}. Now you can directly add any email addresses as well.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc:5]:
```suggestion
You can send email notifications to any user or group of users in {CENTRAL}.
```

File: doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+.Prerequisites
+
+You have created a project in {CENTRAL} and it contains at least one business process asset.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc:9]:
```suggestion
You have created a project in {CENTRAL}, containing at least one business process asset. 
```

File: doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+.Procedure
+
+. Create a business process.
+. In the process designer, select the *Activities* tool from the tool palette.
+. Select *User*.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc:15]: I believe selecting user is not required, we can directly ask the user to drag and drop the user task.

File: doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+. Drag and drop a user task onto the process designer canvas.
+. In the upper-right corner of the screen, click the *Properties* icon.
+. Expand *Implementation/Execution* and next to *Notifications*, click image:getting-started/btn_assign.png[] to open the *Notifications* window.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc:18]:
```suggestion
. Expand *Implementation/Execution* and click image:getting-started/btn_assign.png[] next to *Notifications*, to open the *Notifications* window.
```

File: doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+. Click *Add*.
+. In the *Notifications* window, enter an email address in the *To: email(s)* field to send task notification emails.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc:20]:
```suggestion
. In the *Notifications* window, enter an email address in the *To: email(s)* field.
```

File: doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
++
+You can add multiple email addresses separated by comma.
+. Enter the subject of an email.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc:23]:
```suggestion
. Enter the subject and body of the email.
```

File: doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+. Enter the body of an email.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc:24]:
```suggestion

```

File: doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+. Click *OK*.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc:25]:
```suggestion
. Click *Ok*.
```

File: doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
++
+In the *Notifications* window, you can see the newly added email addresses in the *To: email(s)* column.
+. Click *OK*.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc:28]:
```suggestion
. Click *Ok*.
```

File: doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+. Expand *Implementation/Execution* and next to *Notifications*, click image:getting-started/btn_assign.png[] to open the *Notifications* window.
+. Click *Add*.
+. In the *Notifications* window, enter an email address in the *To: email(s)* field to send task notification emails.
++
+You can add multiple email addresses separated by comma.
+. Enter the subject of an email.
+. Enter the body of an email.
+. Click *OK*.
++
+In the *Notifications* window, you can see the newly added email addresses in the *To: email(s)* column.
```
Comment (hmanwani-rh) [inline doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc:27]:
```suggestion
You can see the added email addresses in the *To: email(s)* column in the *Notifications* window.
```

**Review by hmanwani-rh — COMMENTED** [review]
@kaldesai Please see the comments below and let me know if you have any questions. 
thanks

File: doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc (hunk 6 above)
Comment (LuboTerifaj) [inline doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc:20]: "to send task notification emails." here doesn't sound clear. I would change it to something like: "to set the recipients of the task notification emails ...".

**Review by LuboTerifaj — CHANGES_REQUESTED** [review]
Hello @kaldesai ,

please see my comments inline.

Thanks!

File: doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc (hunk 6 above)
Comment (kaldesai) [reply doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc:20]: Done.

Comment (kaldesai) [reply doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc:20]: Done

File: doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc (hunk 5 above)
Comment (kaldesai) [reply doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc:18]: Done.

File: doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc (hunk 3 above)
Comment (kaldesai) [reply doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc:9]: Done

[the remark this answers is not in the document]
Comment (kaldesai) [reply doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc:9]: @LuboTerifaj, Can you please clarify more what exactly you would like to remove?

[the remark this answers is not in the document]
Comment (kaldesai) [reply doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc]: @LuboTerifaj, can you please clarify more what exactly you would like to remove?

File: doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+.Prerequisites
+
+You have created a project in {CENTRAL} and it contains at least one business process asset.
+
+.Procedure
+
+. Create a business process.
+. In the process designer, select the *Activities* tool from the tool palette.
+. Select *User*.
+. Drag and drop a user task onto the process designer canvas.
```
[the remark this answers is not in the document]
Comment (kaldesai) [reply doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc]: I clubbed there steps and converted into one step as "**Create a user task**". 
I have also added the cross-reference links for more details about creating a user task. 

I applied the same method for the step "**Create a business process.**" 
This will explain and clarify more. 
Can you please review the same? 
Thank you.

File: doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (LuboTerifaj) [reply doc-content/enterprise-only/processes/configuring-emails-in-task-notification-proc.adoc]: There is a statement: "it contains at least one business process asset", and I am not sure, what it should say to user. Can you please clarify it to me? There is no at least one business process asset. There is exactly one business process asset and by clicking on it, a new process is created.

**Review by LuboTerifaj — APPROVED** [review]
It looks good to me now. Thanks @kaldesai
